discord/enums.py

```diff
@@ -1153,6 +1152,16 @@ class SKUType(Enum):
     subscription        = 5
     subscription_group  = 6
 
+class EntitlementType(Enum):
```

Comment:
Also please document it at the bottom of [docs/api/enumes](https://github.com/mccoderpy/discord.py-message-components/blob/developer/docs/api/enums.rst?plain=1)

Reply:
ok